render/persistentvolume.go

```diff
@@ -0,0 +1,90 @@
+package render
+
+import (
+	"github.com/weaveworks/scope/probe/kubernetes"
+	"github.com/weaveworks/scope/report"
+)
+
+// KubernetesVolumesRenderer renders Kubernetes volume components
+var KubernetesVolumesRenderer = MakeReduce(
+	VolumesRenderer(),
+	PodToVolumeRenderer(),
+	PVCToStorageClassRenderer(),
+)
+
+// VolumesRenderer returns renderer
+func VolumesRenderer() Renderer {
+	return volumesRenderer{}
+}
+
+// volumesRenderer is the renderer to render PVC & PV
+type volumesRenderer struct{}
+
+// Render renders the nodes
```

Comment:
This comment is not very helpful - "nodes" is a very generic term. Say a little about what is really being done.